### PR TITLE
CR for Job #20308 - Refine progress.js

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Code Review for Job #20308 - Workitem available at https://www.worklist.net/20308

### CODE REVIEW

**Comment by problem** [comment]
Good to go on this one.

File: examples/progress.js [hunk 1]
```diff
@@ -23,18 +27,19 @@
...
+        BAR_WIDTH = 480,                            // Dimension of SVG in pixels of visible portion (half) of the bar.
+        BAR_HEIGHT = 30,
         BAR_URL = "http://hifi-public.s3.amazonaws.com/images/progress-bar.svg",
-        BACKGROUND_WIDTH = 360,
-        BACKGROUND_HEIGHT = 60,
-        BACKGROUND_URL = "http://hifi-public.s3.amazonaws.com/images/progress-bar-background.svg",
+        BACKGROUND_WIDTH = 540,
+        BACKGROUND_HEIGHT = 90,
+        BACKGROUND_URL = "http://ctrlaltstudio.com/hifi/progress-bar-background.svg",
+        //BACKGROUND_URL = "http://hifi-public.s3.amazonaws.com/images/progress-bar-background.svg",
```
Comment (Atlante45) [inline examples/progress.js]: Did you mean to leave that like this?

Comment (ctrlaltdavid) [inline examples/progress.js]: Yes, once this PR is merged and live I'll get the SVG on S3 updated and do a new PR with the S3 URL.

Comment (Atlante45) [inline examples/progress.js]: gotcha

**Pull request merged**